index.html

```diff
@@ -0,0 +1,178 @@
+      <h1 class="intro_title">
+        CREATIVE COMMONS <br />GLOBAL SUMMIT<br />
+        2023
+      </h1>
+      <p class="intro_bio">
+        A joyful celebraion believing in the value of opennes and sharing
+        creating a positive change with people from over 80 countries is takinng
+        place in October, in Korea.
+      </p>
+      <p class="event_time" id="event-time">2023.10.15(THU) ~ 16(FRI)</p>
+      <p class="location">
+        @ National Mueseum of Korea, Art Center Nabi and more
+      </p>
+    </section>
+    <section class="main">
+      <h2 class="main_header">Main Program</h2>
+      <span class="main_header_underline"></span>
+      <ul class="main_list">
+        <li class="main_list_card">
+          <img
+            class="main_list_img"
+            src="./images/info_icon1.png"
+            alt="info1_pic"
+            srcset=""
+          />
+          <h3 class="main_list_header">Lecture</h3>
+          <span class="main_list_bio">
+            Lorem ipsum dolor sit, amet adipisicing elit.consectetur adipisicing
+          </span>
+        </li>
+        <li class="main_list_card">
+          <img
+            class="main_list_img"
+            src="./images/info_icon2.png"
+            alt="info1_pic"
+            srcset=""
+          />
+          <h3 class="main_list_header">Exhibition</h3>
+          <span class="main_list_bio">
+            Lorem ipsum dolor sit, amet adipisicing elit.consectetur adipisicing    
+          </span>
+        </li>
+        <li class="main_list_card">
+          <img
+            class="main_list_img"
+            src="./images/info_icon3.png"
+            alt="info1_pic"
+            srcset=""
+          />
+          <h3 class="main_list_header">Forum</h3>
+          <span class="main_list_bio">
+            Lorem ipsum dolor sit, amet adipisicing elit.consectetur adipisicing
+        </li>
+        <li class="main_list_card">
+          <img
+            class="main_list_img"
+            src="./images/info_icon4.png"
+            alt="info1_pic"
+            srcset=""
+          />
+          <h3 class="main_list_header">Workshop</h3>
+          <span class="main_list_bio">
+            Lorem ipsum dolor sit, amet adipisicing elit.consectetur adipisicing
+          </span>
+        </li>
+        <li class="main_list_card">
+          <img
+            class="main_list_img"
+            src="./images/info_icon5.png"
+            alt="info1_pic"
+            srcset=""
+          />
+          <h3 class="main_list_header">Share</h3>
+          <span class="main_list_bio">
+            Lorem ipsum dolor sit, amet adipisicing elit.consectetur adipisicing
+          </span>
+        </li>
+      </ul>
+      <button class="m_link">Join in this especial event!</button>
+      <a href="#" class="main_link">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+      <section class="team-page flex" id="Speakers">
+        <h2 class="subtitle">Meet the team</h2>
```

Comment:
- [ ] please kindly add the line under the meet the team like in the original design.


yours             |  original
:-------------------------:|:-------------------------:
![Screenshot (997)](https://user-images.githubusercontent.com/65298048/234504991-946a5f02-979e-4729-b4b6-98df20344bf9.png)  |  ![Screenshot (998)](https://user-images.githubusercontent.com/65298048/234505096-1dc1b1f6-8222-4815-9547-d5ee5c46827b.png)